zerver/webhooks/gitlab/view.py

```diff
@@ -359,10 +359,9 @@ def get_event(request, payload, branches):
         action = payload.get('object_attributes').get('action')
         event = "{} {}".format(event, action)
     elif event == 'Push Hook':
-        if branches is not None:
-            branch = get_branch_name(payload)
-            if branch not in branches.split(','):
-                return None
+        branch = get_branch_name(payload)
+        if branches is not None and branches.find(branch) == -1:
```

Comment:
@showell I agree. I just thought that a previous review on a similar PR applied to this one too, but it didn't. The context wasn't the same, so my bad. Thanks for pointing this out! :)